### ISSUE TITLE
If I run it will it show me gorilla tag's player count?

### ISSUE THREAD

**Issue by sensor0x0** [NtsFranz/Photon-Player-Count#1]
So I was wondering if when I run this if it will show me Gorilla Tag's player count as I am currently developing a discord bot that has similar features to this but a lot more. I believe you are the creator of the User Count bot in the gorilla tag discord server and I was wondering if you could give me any idea on how to get the player count for my discord bot. Thanks!

**Comment by NtsFranz** [comment]
No, this doesn't contain any data from Gorilla Tag. It has some example implementations that you could use to set up a user count bot in your own game.
